githubapp/middleware_logging.go

```diff
@@ -19,12 +19,21 @@ import (
 	"io"
 	"net/http"
 	"regexp"
+	"strconv"
 	"time"
 
 	"github.com/gregjones/httpcache"
 	"github.com/rs/zerolog"
 )
 
+const (
+	HTTPHeaderRateLimit     = "X-Ratelimit-Limit"
+	HTTPHeaderRateRemaining = "X-Ratelimit-Remaining"
+	HTTPHeaderRateUsed      = "X-Ratelimit-Used"
+	HTTPHeaderRateReset     = "X-Ratelimit-Reset"
+	HTTPHeaderRateResource  = "X-Ratelimit-Resource"
```

Comment:
Let's leave these as unexported constants, unless you think it is helpful for this library to expose them.

Reply:
Fixed in https://github.com/palantir/go-githubapp/pull/413/commits/69c805bd707e57d96f914248576ccfa1df13c63b

I don't see a need to expose them.